docs/user/openstack/README.md

```diff
@@ -101,6 +101,92 @@ handle the request (get routes.route.openshift.io)
 FATAL waiting for openshift-console URL: context deadline exceeded
 ```
 
+## Using an External Load Balancer
+
+This documents how to shift from the API VM load balancer, which is
+intended for initial cluster deployment and not highly available, to an
+external load balancer.
+
+The load balancer must serve ports 6443, 443, and 80 to any users of
+the system.  Port 49500 is for serving ignition startup configurations
+to the OpenShift nodes and should not be reachable outside of the cluster.
+
+The first step is to add floating IPs to all the master nodes:
+
+* `openstack floating ip create --port master-port-0 <public network>`
+* `openstack floating ip create --port master-port-1 <public network>`
+* `openstack floating ip create --port master-port-2 <public network>`
+
+Once complete you can see your floating IPs using:
+
+* `openstack server list`
+
+These floating IPs can then be used by the load balancer to access
+the cluster.  An example haproxy configuration for port 6443 is below.
+The other port configurations are identical.
+
+```
+listen <cluster name>-api-6443
+    bind 0.0.0.0:6443
+    mode tcp
+    balance roundrobin
+    server <cluster name>-master-2 <floating ip>:6443 check
+    server <cluster name>-master-0 <floating ip>:6443 check
+    server <cluster name>-master-1 <floating ip>:6443 check
+```
+
+Next step is to allow access to the network the load balancer is on:
+
+* `openstack security group rule create master --remote-ip <load balancer CIDR> --ingress --protocol tcp --dst-port 6443`
+* `openstack security group rule create master --remote-ip <load balancer CIDR> --ingress --protocol tcp --dst-port 443`
+* `openstack security group rule create master --remote-ip <load balancer CIDR> --ingress --protocol tcp --dst-port 80`
+
+You could
+also specify a specific IP address with /32 if you wish.
```

Comment:
It probably doesn't matter when this gets rendered, but this could be combined into 1 line.